### ISSUE TITLE
Performance on android is slow :(

### ISSUE THREAD

**Issue by waleedarshad-vf** [oblador/react-native-lightbox#57]

**Comment by vedmant** [comment]
Yeah, it takes about 20 seconds to open lightbox on android when I test with Expo, I see every frame, fame time is more than 1 second. That's unusable at all.

**Comment by SergioReis97** [comment]
For anyone stumbling into this, try to go to react native developer menu and disable debugging (shacking the device or in the case of an emulator ctrl+m). I've had the issue of some animations being slow while debugging and this one appears to be due to the same reasons. I don't consider this a major issue since this will never happen on production builds
